.github/workflows/ci_windows.yml

```diff
@@ -0,0 +1,37 @@
+name: CI_windows
+
+on: [push, pull_request]
+
+env:
+  CI: "ON"
+
+jobs:
+  Build:
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - run: cmake -G "MinGW Makefiles" -DCMAKE_SH="CMAKE_SH-NOTFOUND" -Wdev -B build
+      env:
+        FC: gfortran
+        CC: gcc
+        CXX: g++
+
+    - name: CMake build
+      run: cmake --build build --parallel
+
+    - run: cmake --build build --verbose --parallel 1
+      if: failure()
+
+    - name: CTest
+      run: ctest --parallel -V
+      working-directory: build
+
+    - uses: actions/upload-artifact@v1
+      if: failure()
+      with:
+        name: WindowsCMakeTestlog
+        path: build/Testing/Temporary/LastTest.log
```

Comment:
I can consolidate this with the build matrix once this is merged so that it's not a separate workflow. I'll have to look into the windows runners some more. If it is possible to test multiple GFortrans that would be nice but any windows tests are a win in my book.